gc/base/standard/Scavenger.cpp

```diff
@@ -1543,6 +1546,33 @@ MM_Scavenger::copy(MM_EnvironmentStandard *env, MM_ForwardedHeader* forwardedHea
 			scavStats->_flipBytes += objectCopySizeInBytes;
 			scavStats->getFlipHistory(0)->_flipBytes[oldObjectAge + 1] += objectReserveSizeInBytes;
 		}
+		if (_extensions->scavengerScanOrdering == MM_GCExtensions::OMR_GC_SCAVENGER_SCANORDERING_DYNAMIC_BREADTH_FIRST) {
+			UDATA hotFieldOffset = _extensions->objectModel.getHotFieldOffset(forwardedHeader);
```

Comment:
Need to have OMR proxy of getHotFieldOffset in ObjectModelBase class. It's only defined in OpenJ9.